propka/run.py

```diff
@@ -36,34 +36,68 @@ def main(optargs=None):
             my_molecule.write_propka()
 
 
-def single(pdbfile, optargs=None):
-    """Run a single PROPKA calculation using *pdbfile* as input.
+def single(filename: str, optargs: list, stream=None, write_pka: bool = True):
```

Comment:
default argument for `optargs` removed by accident? Should either be `optargs = None`, or the documentation should not say that it's optional.

Reply:
Yep that was a typo, fixed in aad75f1